bin/p2-dsctl/main.go

```diff
@@ -60,6 +61,14 @@ var (
 	updateMinHealth = cmdUpdate.Flag("minhealth", "The minimum health of the daemon set").String()
 	updateName      = cmdUpdate.Flag("name", "The cluster name (ie. staging, production)").String()
 	updatePodID     = cmdUpdate.Flag("pod", "The pod ID on the daemon set").String()
+
+	cmdOverrideSelector = kingpin.Command(CmdOverrideSelector, `
+		The raw selector for the daemon set represented as a string.
```

Comment:
Could you change this help text to describe what the command does? Arguments already have their own descriptions below.